dynamic-filter-query.sql

```diff
@@ -2,137 +2,145 @@ DECLARE fromDate timestamp DEFAULT CAST(DATE_SUB(CURRENT_DATE(), INTERVAL 1 DAY)
 DECLARE toDate timestamp DEFAULT CAST(CURRENT_DATE() AS timestamp);
 DECLARE defaultFilter numeric DEFAULT 0.05;
 DECLARE minRequests int64 DEFAULT 5000;
-DECLARE minBidRate numeric DEFAULT 0.1;
-DECLARE minAverageBid numeric DEFAULT 0.1;
-DECLARE minCombinedBidRate numeric DEFAULT 0.3;
-DECLARE minCombinedAverageBid numeric DEFAULT 0.2;
-
+DECLARE minBidRate numeric DEFAULT 0.25;
+DECLARE minAverageBid numeric DEFAULT 0.2;
+DECLARE minCombinedBidRate numeric DEFAULT 0.4;
+DECLARE minCombinedAverageBid numeric DEFAULT 0.3;
 
 WITH continentEntries AS (
 
-SELECT partner Bidder, geo_continent Continent,
-Sum(bids) Bids,
-Sum(bids) + Sum(no_bids) Total,
-COALESCE(SAFE_DIVIDE(SUM(avg_bid_cpm*bids), SUM(bids)), 0) Average_Bid
-FROM `streamamp-production.snapshots.pbs_a_daily`
-where error_code IS NULL
-AND geo_continent IS NOT NULL
-AND geo_continent != 'unknown'
-AND partner IS NOT NULL
-AND time >= fromDate AND time < toDate
-GROUP BY partner, geo_continent
+    SELECT bidder_id Bidder, geo_continent Continent,

```

Comment:
Could we add a small comment above each query describing the data is pulling, please? 🤕